atomic_reactor/plugins/post_group_manifests.py

```diff
@@ -208,6 +209,36 @@ def build_list(self, manifests):
                 ],
         }, indent=4)
 
+    def check_existing_vr_tag(self):
```

Comment:
maybe docstring that even explains what vr_tag is could be useful here.

Reply:
sure